lib/ledger_sync/ledgers/request.rb

```diff
@@ -16,14 +16,15 @@ def initialize(args = {})
         @method = args.fetch(:method, nil)
         @params = args.fetch(:params, {})
         @url = args.fetch(:url, nil)
+        @faraday_client = args.fetch(:faraday_client, Faraday.new)
```

Comment:
Should this be `Faraday.new` or `Faraday`? The old code would suggest `Faraday` but maybe these are the same. Can you verify?

Reply:
They're not the same. `Faraday.new` creates a `connection` (don't like the term but this is what [Faraday docs](https://lostisland.github.io/faraday/usage/) use) that uses Faraday's default settings as they were at that point in time. 

```ruby
Faraday.default_adapter = Faraday::Adapter::Typhoeus
Faraday.adapter
=> Faraday::Adapter::Typhoeus

connection = Faraday.new
connection.adapter
=> Faraday::Adapter::Typhoeus

Faraday.default_adapter = :test
Faraday.adapter
=> Faraday::Adapter::Test

connection.adapter
=> Faraday::Adapter::Typhoeus
```

If you want to keep backwards compatibility to 100%, you'd use `Faraday` instead of `Faraday.new`.
But the only difference it makes is if you dynamically change Faraday default configuration while running the app (instead of just once when you initialise it), which is very problematic on its own.

We could also opt to use `connection` rather than `faraday_client` as the variable name, if we want to stay more inline with Faraday's own terminology.

Reply:
To confirm `Faraday.new` accepts `post` or other method names just like `Faraday`?

Reply:
@ryanwjackson that's correct